staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -162,6 +163,26 @@ func (r *Request) Suffix(segments ...string) *Request {
 	return r
 }
 
+// AsTable adds the "as=Table;v=v1alpha1;g=meta.k8s.io" parameter to the Accept header
+func (r *Request) AsTable(gv metav1.GroupVersion) *Request {
```

Comment:
@smarterclayton this looks like the correct spot to set the options, since I wouldn't want an entirely new client for one request.  The rest of the pull seems to flow from here.  In the builder, we could argue things like enum over bool and the like, but given how `get.go` works, we need to set the value from `pkg/kubectl/resource/helper.go`

Reply:
I don't think request should know anything about Table.  This should just be SetHeader, only higher levels know about Tables.  PartialObjectMetadata will also come through this path and I don't want `AsPartialObjectMetadata`